### PR TITLE
Make services print on a single line.

### PR DESCRIPTION
**Pull request by brendandburns**
Fixes https://github.com/GoogleCloudPlatform/kubernetes/issues/9329

### CODE REVIEW

**Comment by bgrant0607** [comment]
Thanks. Assigning to @bprashanth, who wrote this example.

**Comment by bprashanth** [comment]
LGTM

File: pkg/kubectl/resource_printer.go [hunk 1]
```diff
@@ -259,7 +259,7 @@ func (h *HumanReadablePrinter) HandledResources() []string {
 var podColumns = []string{"NAME", "READY", "STATUS", "RESTARTS", "AGE"}
 var podTemplateColumns = []string{"TEMPLATE", "CONTAINER(S)", "IMAGE(S)", "PODLABELS"}
 var replicationControllerColumns = []string{"CONTROLLER", "CONTAINER(S)", "IMAGE(S)", "SELECTOR", "REPLICAS", "AGE"}
-var serviceColumns = []string{"NAME", "LABELS", "SELECTOR", "IP(S)", "PORT(S)", "AGE"}
+var serviceColumns = []string{"NAME", "CLUSTER IP", "EXTERNAL IP", "PORT(S)", "SELECTOR", "AGE"}
```
Comment (bgrant0607) [inline pkg/kubectl/resource_printer.go]: In order for cut, etc. to work, we need a uniform number of columns, so please use CLUSTER_IP and EXTERNAL_IP.

**Comment by bgrant0607** [comment]
There's apparently a test that the code does what it does:
```
--- FAIL: ExamplePrintServiceWithNamespacesAndLabels (0.00s)
got:
|NAMESPACE NAME CLUSTER IP EXTERNAL IP PORT(S) SELECTOR AGE L1|
|ns1 svc1 10.1.1.1 unknown 53/UDP,53/TCP s=magic 10y value|
|ns2 svc2 10.1.1.2 unknown 80/TCP,8080/TCP s=kazam 10y dolla-bill-yall|
||
want:
|NAMESPACE NAME LABELS SELECTOR IP(S) PORT(S) AGE L1|
|ns1 svc1 l1=value s=magic 10.1.1.1 53/UDP 10y value|
| 53/TCP |
|ns2 svc2 l1=dolla-bill-yall s=kazam 10.1.1.2 80/TCP 10y dolla-bill-yall|
| 8080/TCP |
||
FAIL
```

**Comment by bprashanth** [comment]
I remember now why I reached into json. That feels better than having to ask a user to bucket ips based on rfc 1918. I don't know if the 10. bucketing trick will work on all clouds, and most people don't know the difference between a 10. and a public ip anyway so it's just as mysterious.

**Comment by a-robinson** [comment]
The shippable results seem to be stale. I'll re-kick it but it doesn't look like @brendandburns has re-pushed since the broken test was mentioned?

I'm also assuming we don't care about all the other examples/docs whose sample output is now out of date?

File: pkg/kubectl/resource_printer.go (hunk 1 above)
Comment (brendandburns) [inline pkg/kubectl/resource_printer.go]: done.

**Comment by bgrant0607** [comment]
I mainly care about updating examples that otherwise extract the ingress ip from json or yaml, such as:

```
examples/k8petstore/k8petstore-loadbalancer.sh:    PUBLIC_IP=$($kubectl get services/frontend --namespace=$NS -o template --template="{{range .status.loadBalancer.ingress}}{{.ip}}{{end}}")
examples/phabricator/README.md:$ kubectl get services phabricator -o template --template='{{(index .status.loadBalancer.ingress 0).ip}}{{"\n"}}'
```

**Comment by brendandburns** [comment]
Comments addressed.  Test fixed. Lots of documents updated.  (I left a couple of places where templates were being used to set shell variables, which I think is legitimate)

ptal.

Thanks!
--brendan

File: docs/getting-started-guides/gce.md [hunk 2]
```diff
@@ -145,15 +145,11 @@ $ kubectl get --all-namespaces services
...
-kube-system   monitoring-heapster   kubernetes.io/cluster-service=true,kubernetes.io/name=Heapster                   k8s-app=heapster        10.0.59.48     80/TCP
-kube-system   monitoring-influxdb   kubernetes.io/cluster-service=true,kubernetes.io/name=InfluxDB                   k8s-app=influxGrafana   10.0.210.156   8083/TCP
-                                                                                                                                                            8086/TCP
+NAMESPACE     NAME                  CLUSTER_IP       EXTERNAL_IP       PORT(S)       SELECTOR               AGE
+default       kubernetes            10.0.0.1         <none>            443/TCP       <none>                 1d
+kube-system   kube-dns              10.0.0.2         <none>            53/TCP,53/UDP k8s-app=kube-dns       1d
+kube-system   kube-ui               10.0.0.3         <none>            80/TCP        k8s-app=kube-ui        1d
+...                                                                                                                                                            8086/TCP
```
Comment (bgrant0607) [inline docs/getting-started-guides/gce.md]: 8086/TCP looks like leftover cruft.

**Pull request merged**